codalab/client/local_bundle_client.py

```diff
@@ -1339,7 +1339,8 @@ def get_user_info(self, user_id, fetch_extra=False):
     def update_user_info(self, user_info):
         user_id = self._current_user_id()
         is_root = (user_id == self.model.root_user_id)
-        is_user = (user_id == user_info['user_id'])
+        if 'user_id' not in user_info:
```

Comment:
If `user_id` is not specified, we assume that it is the current user.

The original problem was that `LocalBundleClient.get_user_info()` would create a new user with the given user_id if one did not exist yet.  (This is especially problematic if the username is specified at the command-line instead of the user ID, creating a user with that username as the user ID.) This is no longer necessary after the user DB migration so I've removed the `get_user_info()` call.